src/routes/router.ts

```diff
@@ -34,6 +44,14 @@ router.get('/studentGroups/:id', findStudentGroup);
 router.put('/studentGroups/:id', updateStudentGroup);
 router.delete('/studentGroups/:id', deleteStudentGroup);
 
+router.post('/permissions/user', createUserPermission);
+router.put('/permissions/user/:id', updateUserPermission);
+router.delete('/permissions/user/:id', deleteUserPermission);
+
+router.post('/permissions/studentGroup', createStudentGroupPermission);
+router.put('/permissions/studentGroup/:id', updateStudentGroupPermission);
+router.delete('/permissions/studentGroup/:id', deleteStudentGroupPermission);
```

Comment:
could we simplify this to
```suggestion
router.post('/permissions/group', createStudentGroupPermission);
router.put('/permissions/group/:id', updateStudentGroupPermission);
router.delete('/permissions/group/:id', deleteStudentGroupPermission);
```

And i'm not sure, should it be plural too? `permissions/studentGroups/:id` or `permissions/groups/:id`?